src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  margin: 0;
+  padding: 30px 14px 14px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.product__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0 0 5px;
+}
+
+.product__code {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0 0 15px;
+}
+
+.product__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 15px;
+  margin: 0 0 20px;
+}
+
+.product__stars {
+  display: flex;
+  align-items: center;
+}
+
+.product__stars .star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.product__stars .star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
add rules for another cases, so i could change class and count of active stars also changes

Reply:
added